py/desiutil/test/test_bitmask.py

```diff
@@ -6,8 +6,9 @@
                         print_function, unicode_literals)
 # The line above will help with 2to3 support.
 import unittest
-from ..bitmask import BitMask, _MaskBit
+from desiutil.bitmask import BitMask, _MaskBit
```

Comment:
I don't understand what you mean by "making a test run standalone".  Also, this is not actually an absolute import.  This is just poor import style.